interactions/api/models/guild.py

```diff
@@ -236,6 +241,422 @@ def __init__(self, **kwargs):
+                guild_id=int(self.id),
+                user_id=member_id,
+                role_id=roles,
+                reason=reason,
+            )
+
+    async def remove_member_roles(
+        self,
+        roles: Union[List[Role], Role, int],
+        member_id: int,
+        reason: Optional[str],
+    ) -> None:
+        """
+        This method removes a or multiple role(s) from a member
+        :param roles: The role(s) to remove. Either ``Role`` object or role_id
+        :type roles: Union[List[Role], Role, int]
+        :param member_id: The id of the member to remove the roles from
+        :type member_id: int
+        :param reason?: The reason why the roles are removed
+        :type reason: Optional[str]
+        """
+        if isinstance(roles, list):
+            roles = [int(role.id) for role in roles]
+            for role in roles:
+                await self._client.remove_member_role(
+                    guild_id=int(self.id),
+                    user_id=member_id,
+                    role_id=role,
+                    reason=reason,
+                )
+        elif isinstance(roles, Role):
+            await self._client.remove_member_role(
+                guild_id=int(self.id),
+                user_id=member_id,
+                role_id=int(roles.id),
+                reason=reason,
+            )
+        else:
+            await self._client.remove_member_role(
+                guild_id=int(self.id),
+                user_id=member_id,
+                role_id=roles,
+                reason=reason,
+            )
+
+    async def create_guild_role(
+        self,
+        name: str,
+        # permissions,
+        color: Optional[int] = 0,
+        hoist: Optional[bool] = False,
+        # icon,
+        # unicode_emoji,
+        mentionable: Optional[bool] = False,
+        reason: Optional[str] = None,
+    ) -> Role:
+        """
+        Creates a new role in the guild
+        :param name: The name of the role
+        :type name: str
+        :param color?: RGB color value as integer, default ``0``
+        :type color: Optional[int]
+        :param hoist?: Whether the role should be displayed separately in the sidebar, default ``False``
+        :type hoist: Optional[bool]
+        :param mentionable?: Whether the role should be mentionable, default ``False``
+        :type mentionable: Optional[bool]
+        :param reason?: The reason why the role is created, default ``None``
+        :type reason: Optional[str]
+        :return: The created Role
+        :rtype: Role
+        """
+
+        payload = Role(
+            name=name,
+            color=color,
+            hoist=hoist,
+            mentionable=mentionable,
+        )
+        res = await self._client.create_guild_role(
+            guild_id=int(self.id),
+            reason=reason,
+            data=payload._json,
+        )
+        role = Role(**res, _client=self._client)
+        return role
+
+    async def get_guild_member(
```

Comment:
A problem I see with this for consistency purposes is having some methods with `get_guild_X` and others as `get_X` when all endpoints for `Guild` are always in reference to the guild object according to the Discord Developers documentation. This use case is `ctx.guild.get_guild_member()` which I think is unnecessary because calling this off of a guild object is inclusive enough.

I would recommend calling this `get_member` instead.